build.xml

```diff
@@ -71,6 +71,7 @@
     <target name="test" depends="compile, test-compile">
         <java classname="${test.mainclass}" fork="true" failonerror="true">
             <jvmarg value="-Xmx4g" />
+            <jvmarg value="-ea" />
```

Comment:
Maybe leave a comment that this is needed for correctness

Reply:
Will do.

Reply:
Done.